### PR TITLE
ci: run fuzzers

### PR DESCRIPTION
**Pull request by dmehala**
# Description
Compile and run fuzzers on main branch for 5min.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,6 +24,26 @@ executors:
...
+        type: string
+      description:
+        type: string
+    steps:
+      - run:
+          name: << parameters.description >>
+          command: |
+            set +e
+            timeout << parameters.duration >> << parameters.command >>
+            if [[ $? -eq 124 ]]; then
```
Comment (dmehala) [inline .circleci/config.yml:43]:
```suggestion
            # `124` is the exit code timeout returns when the duration is reached
            if [[ $? -eq 124 ]]; then
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -171,6 +202,25 @@ jobs:
...
+      MAKE_JOB_COUNT: 8
+    steps:
+      - checkout
+      - run: bin/with-toolchain llvm cmake . -B .build -DCMAKE_BUILD_TYPE=Debug -DDD_TRACE_BUILD_FUZZERS=1 -DDD_TRACE_ENABLE_SANITIZE=1
+      - run: cmake --build .build -j ${MAKE_JOB_COUNT} --target dd_trace_cpp-fuzzers
+      - run_timeout: 
+          description: Run W3C propagation fuzzer
+          command: ./.build/fuzz/w3c-propagation/w3c-propagation-fuzz
+          duration: 5m
+      - run:
```
Comment (pablomartinezbernardo) [inline .circleci/config.yml:219]: Any reason not to use `run_timeout` here? Or maybe merge mistake?

**Review by pablomartinezbernardo — APPROVED** [review]
Just one small comment added

File: .circleci/config.yml (hunk 2 above)
Comment (dmehala) [reply .circleci/config.yml:219]: That's a mistake. Thank you!

Comment (dmehala) [reply .circleci/config.yml:219]:
```suggestion
      - run_timeout: 
```

**Pull request merged**